src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/infrastructure/client/util/HomeOfficeApiUtil.java

```diff
@@ -38,6 +38,21 @@ public HttpHeaders getHomeOfficeHeader() {
         return ImmutableMap.of("code", "HMCTS", "description", "HM Courts and Tribunal Service");
```

Comment:
It's not a good idea to call a class *Util.  It's often a smell that the class does not have a singular purpose and is mixing different things.  So please rename this class and break out anything that has different purposes into their own classes.

Reply:
This will be addressed in #77